qiime2/sdk/tests/test_usage.py

```diff
@@ -380,6 +390,10 @@ def test_init_data(self):
         with self.assertRaisesRegex(TypeError, 'expected Metadata'):
             use.init_metadata('name', lambda: object)
 
+        with self.assertRaisesRegex(ValueError, 'expected a ScopeRecord.'):
+            use.init_data_collection('', list, object)
+            use.init_data_collection('', list, object, object)
+
```

Comment:
Will this do the trick, @thermokarst?

Reply:
For checking the type validation of the inputs? Yes I think so! Although, you'll need to tweak a bit. First, you can't put both failing calls to `init_data_collection` in one `self.assertRaisesRegex` context: the second call to `init_data_collection` will never run (because the first raises an exception. Second, the multiple-object example might be more useful if it has a ScopeRecord first, then an object, to test that we are checking all the values:

```suggestion
        with self.assertRaisesRegex(ValueError, 'expected a ScopeRecord.'):
            use.init_data_collection('', list, object)
        with self.assertRaisesRegex(ValueError, 'expected a ScopeRecord.'):
            use.init_data_collection('', list, ScopeRecord(...), object)
```